packages/contracts/gasTest/gasCalc.js

```diff
@@ -1175,48 +1163,15 @@ contract('Gas cost tests', async accounts => {
     th.appendData(gasResults, message, data)
   })
 
-  // it("", async () => {
-  //   const message = 'provideToSP(), No pending rewards, deposit all issued LUSD: all accounts withdraw 180 LUSD, make second deposit, provide 90 LUSD'
-  //   await th.openTrove_allAccounts(_10_Accounts, contracts, dec(10, 'ether'), 0)
-  //   await th.withdrawLUSD_allAccounts(_10_Accounts, contracts, dec(130, 18))
-  //   await th.provideToSP_allAccounts(_10_Accounts, stabilityPool, dec(50, 18))
-
-  //   // >> FF time and one account tops up, triggers LQTY gains for all
-  //   await th.fastForwardTime(timeValues.SECONDS_IN_ONE_HOUR, web3.currentProvider)
-  //   await stabilityPool.provideToSP(dec(1, 18), { from: _10_Accounts[0] })
-
-  //   // Check the other accounts have LQTY gain
-  //   for (account of _10_Accounts.slice(1)) {
-  //     const LQTYGain = await stabilityPool.getDepositorLQTYGain(account)
-  //     assert.isTrue(LQTYGain.gt(toBN('0')))
-  //   }
-
-  //   await th.fastForwardTime(timeValues.SECONDS_IN_ONE_HOUR, web3.currentProvider)
-
-  //   // top-up of StabilityPool Deposit
-  //   const gasResults = await th.provideToSP_allAccounts(_10_Accounts, stabilityPool, dec(50, 18))
-  //   th.logGasMetrics(gasResults, message)
-  //   th.logAllGasCosts(gasResults)
-
-  //   th.appendData(gasResults, message, data)
-  // })
-
   it("", async () => {
```

Comment:
we've the only assertion in this test, what are we testing now?

or is this one of those cases where you mentioned the tests that come after expect the prior ones to have run?

Reply:
yeah this was the weird dependancy where if its combined into the next test it fails, if its removed the next test fails. It requires more investigation at some point